gap/PolygonalComplexes/animating.gd

```diff
@@ -0,0 +1,537 @@
+#!
+#! After seting parameters of the vertices, it is necessary to set parameters of the edges.
+#! We demonstrate the problem of outdated parameters
+#! by an example, where we set coordinates
+#! and redefine these mirrored at Zero.
+#! @BeginExampleSession
+#! gap> verticesPositions := [
+#! > [ 1, -1/Sqrt(3.), -1/Sqrt(6.) ],
+#! > [ -1, -1/Sqrt(3.), -1/Sqrt(6.) ],
+#! > [ 0, 2/Sqrt(3.) , -1/Sqrt(6.) ],
+#! > [ 0, 0, 3/Sqrt(6.) ] ];;
+#! gap> verticesPositionsMirrored := -verticesPositions;;
+#! gap> tet := Tetrahedron();;
+#! gap> printRecord := SetVertexCoordiantes3D(tet, verticesPositions, rec());;
+#! gap> printRecord := CalculateParametersOfEdges(tet, printRecord);;
+#! gap> # coordinates of the center of the edge (1,2):
+#! > printRecord.edges[EdgeBetweenVertices(tet, 1, 2)][1];
+#! [ 0., -0.57735, -0.408248 ]
+#! gap> printRecord := SetVertexCoordiantes3D(tet, verticesPositionsMirrored,
+#! > printRecord);;
+#! gap> # coordinates of the edge (1,2) did not change:
+#! > printRecord.edges[EdgeBetweenVertices(tet, 1, 2)][1];
+#! [ 0., -0.57735, -0.408248 ]
+#! gap> DrawSurfaceToJavaScriptCalculate(tet,
+#! > "doc/Tetrahedron_PositionMirrored.html", printRecord, false);;
+#! @EndExampleSession
+#! @InsertChunk Example_OctahedronPositionMirrored
+#! The function <K>DrawSurfaceToJavaScript</K> always calculates the parameters of the edges depending on the current vertices positions.
+#! If you want to avoid that, you can use the function <K>DrawSurfaceToJavaScriptCalculate</K> and set the last parameter to false. In this case no new parameters are computed except that no parameters were ever set.
+
+#! @BeginGroup SetVertexCoordinates3D
+#! @Description
+#! Save the given list of 3D-coordinates in the given or the empty print record. If the format of the 3D-coordinates (3D-coordinates have to be a list of 3 entries of floats) is not correct, then an error is shown.
+#! The NC-version does not check the coordinate format.
+#! @Returns the updated print record.
+#! @Arguments surface, newCoordinatesList[, printRecord]
+DeclareOperation( "SetVertexCoordiantes3D", [IsSimplicialSurface and IsNotEdgeRamified, IsDenseList, IsRecord] );
+#! @Arguments newCoordinatesList[, printRecord]
+DeclareOperation( "SetVertexCoordiantes3DNC", [IsDenseList, IsRecord] );
+#! @EndGroup
+
+#! @BeginGroup GetVertexCoordinates3D
+#! @Description
+#! Extract the 3D-coordinates from the print record of the vertex in position <A>i</A> of <K>Vertices</K> from the corresponding <A>surface</A>. If the format of the 3D-coordinates (3D-coordinates have to be a list of 3 entries of floats) is not correct, then an error is shown. This can happen, if the NC version is used to store the 3D-coordinates.
```

Comment:
.. of the vertex <A>i</A> from <A>surface</A>.

The label of the vertex is its name. You could also change i to v to make this a bit easier to parse.